test/public-stats.test.js

```diff
@@ -228,7 +228,7 @@ describe('public-stats', () => {
       const honestMeasurements = [
         { ...VALID_MEASUREMENT },
         // HTTP_NOT_ADVERTISED means the deal is indexed
-        { ...VALID_MEASUREMENT, cid: 'bafy2', indexerResult: 'HTTP_NOT_ADVERTISED', retrievalResult: 'IPNI_HTTP_NOT_ADVERTISED' },
+        { ...VALID_MEASUREMENT, cid: 'bafy2', indexerResult: 'HTTP_NOT_ADVERTISED', retrievalResult: 'ERROR_502' },
```

Comment:
`IPNI_HTTP_NOT_ADVERTISED` is not a valid retrieval result code. The indexer result `HTTP_NOT_ADVERTISED ` means that only Graphsync was advertised, and the retrieval check continues to retrieve the CAR bytes.